src/main/java/com/hcl/appscan/jenkins/plugin/auth/ASoCCredentials.java

```diff
@@ -24,23 +24,37 @@ public class ASoCCredentials extends UsernamePasswordCredentialsImpl {
 
 	private static final long serialVersionUID = 1L;
 	private Secret m_token;
+	public String m_url;
+    public boolean m_certificates;
```

Comment:
I'm guessing this option is meant to indicate the server has an untrusted certificate (e.g. self-signed). Can the name be updated to be more descriptive? For example, m_acceptInvalidCerts or something like that.

Reply:
Okay Matt, I will use "m_acceptInvalidCerts".